examples/robust_imports/src/import_coverage/import_coverage.ts

```diff
@@ -0,0 +1,152 @@
+import $icQuery, * as ic from './types';
+import { text, text as mT, text as mDT, text as mCT } from './types';
```

Comment:
I'm not sure if having these multiple aliased types are actually necessary. Having the single text as mT is probably enough... but you do you

Reply:
I don't remember why I decided to break them up into multiple imports. It's possible I was just rename happy and there is no good reason. But maybe there is a reason. On possible reason is just to make sure our generated Symbol Table has more than one entry in the text list? That seems pretty trivial. But since it's not hurting anyone and I don't remember why I did it, I am inclined to leave it there.